docker/.env

```diff
@@ -33,21 +31,39 @@ REDIS_PASSWORD=infini_rag_flow
 
 SVR_HTTP_PORT=9380
 
-RAGFLOW_IMAGE=infiniflow/ragflow:dev-slim
+RAGFLOW_IMAGE=infiniflow/ragflow:dev-slim # slim version docker image
```

Comment:
RAGFLOW_IMAGE=infiniflow/ragflow:dev-slim # the Docker image for the slim version